spec.bs

```diff
@@ -372,10 +370,8 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
     Issue: This probably needs to be its own algorithm, how is "most recent" defined here?
 
  1. Let |maxSize| be |issuer|'s [=max batch size=] or the global limit of 100, whichever is smaller.
```

Comment:
So what is the difference between |maxSize| (which I added, I think) and |numTokens| now?

Reply:
Missed that, they are identical. I've removed maxSize since it doesn't seem used anywhere?